src/main/java/org/folio/circulation/infrastructure/storage/loans/LoanPolicyRepository.java

```diff
@@ -85,6 +85,16 @@ public CompletableFuture<Result<Loan>> findPolicyForLoan(Loan loan) {
         .thenApply(result -> result.map(loan::withLoanPolicy));
   }
 
+  private CompletableFuture<Result<LoanPolicy>> getLoanPolicy(LoanAndRelatedRecords relatedRecords, Loan loan) {
```

Comment:
No need to pass both relatedRecords and loan. One line above calling this method, we're getting loan from relatedRecords:
Result.of(relatedRecords::getLoan)